packages/common/src/components/Tags/index.tsx

```diff
@@ -9,6 +9,7 @@ import Tag from './Tag';
 type Props = {
   tags: Array<string>;
```

Comment:
@SaraVieira This can be

```suggestion
   tags: string[];
```